tests/data/test_dataloader.py

```diff
@@ -503,11 +513,19 @@ def test_finetuning_dataloader_custom_split_remote(
         tokenizer_kwargs={'model_max_length': max_seq_len},
     )
 
+    args_list = []
```

Comment:
Can you use mock [patching](https://docs.python.org/3/library/unittest.mock.html#patch) instead to check the arguments its called with?

Reply:
I _think_ it might be something like
```
with patch('llmfoundry.data.finetuning.dataloader.get_file', wraps=mock_get_file) as m_get_file:
   m_get_file.assert_called_with(...)
```